### PR TITLE
Update the README.md file to include main features and screenshots.

### PR DESCRIPTION
**Pull request by zpChris**
I updated the `README.md`file, and include screenshots in a `site_screenshots` folder in this repository in order to include the images in the `README.md` file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,50 @@
...
 
 This is the Google STEP capstone project by Chris, Adam, and Eshika. 
-**T**ravel **I**nformation **S**ystem is meant to fascilate travel planning.
+TravIS, the **T**ravel **I**nformation **S**ystem, is meant to facilitate travel planning, allowing you to plan the optimal trip.
 
-Visit the website at http://offroad-pod-step-2020.appspot.com.
+Visit the website at http://offroad-pod-step-2020.appspot.com. (Note: This link may become defunct at some point.)
+
+## Main Features
+1. Create a new trip by adding a destination (hotel), start and end dates, and points of interest (POIs) they would like to visit. TravIS will calculate the optimal route and itinerary for the user's trip.
```
Comment (bheff-work) [inline README.md:10]: nit: instead of saying "they", use "the user" to match the style/tense of the rest of the features list

File: README.md [hunk 2]
```diff
@@ -1,7 +1,50 @@
...
+    - TravIS will also provide suggested points of interest based on the user's destination.
+2. View the itinerary and route directions for each day through an embedded Google Maps widget on the Maps page.
+    - TravIS reorders the POIs entered by the user and calculates a daily route that is optimized for travel time using the Maps API. This route starts and ends at the user's hotel and includes the user's desired POIs as stops along the way.
+3. See the overall trip schedule, different calendar views on the Calendar page.
```
Comment (bheff-work) [inline README.md:14]: nit: instead of the comma, I'd put 'and'

Comment (zpChris) [reply README.md:14]: Fixed!

File: README.md (hunk 1 above)
Comment (zpChris) [reply README.md:10]: Done -- thanks!

**Pull request merged**